code_examples/ApacheKafka/prototype/README.md

```diff
@@ -0,0 +1,54 @@
+
+# Apache Kafka Stream Data Integration Prototype
+
+This prototype demonstrates SemWES' capabilities to integrate data streams using the popular distributed streaming platform [Apache Kafka](https://kafka.apache.org/) .
+
+The examplary setup consists of 3 components: 
+ - [Kafka Server/Broker](kafka_server/README.md), stores all stream data and provides interfaces to access them
+ - [Kafka Services](kafka_services/README.md), simple producer and consumer web services to interact with the data streams
+ - [Kafka sh](kafka_sh/README.md), a shell instance to manually test all components
+
+All components are deployed using docker and are connected to the same docker network "semwes_kafka", to easily and securely communicate with each other.
+
+To ease the setup, a set of ssh keys and passwords has already been created, details can be found in a [dedicated README](ssl/README.md). If you use this prototype in production, please regenerate all keys and register them with the respective components, as shown in the previously mentioned ReadMe.
```

Comment:
I find it dangerous to include passwords here, users might just ignore the instruction to re-generate them.